ios/FluentUI/Tooltip/TooltipViewController.swift

```diff
@@ -40,51 +40,45 @@ class TooltipView: UIView {
             titleLabel.textAlignment = textAlignment
         }
 
-        addSubview(textContainer)
+        view.addSubview(textContainer)
 
         // Shadow
-        layer.insertSublayer(CALayer(), at: 0)
-        layer.insertSublayer(CALayer(), at: 0)
+        view.layer.insertSublayer(CALayer(), at: 0)
+        view.layer.insertSublayer(CALayer(), at: 0)
         updateShadows(tokenSet: tokenSet)
     }
 
     required init?(coder aDecoder: NSCoder) {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
-    override func layoutSubviews() {
-        super.layoutSubviews()
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
 
-        backgroundView.frame = bounds
-
-        if positionController.arrowDirection.isVertical {
-            arrowImageView.frame.origin.x = positionController.arrowPosition
-            backgroundView.frame.size.height -= arrowImageView.frame.height
-        } else {
-            arrowImageView.frame.origin.y = positionController.arrowPosition
-            backgroundView.frame.size.width -= arrowImageView.frame.width
+        coordinator.animate(alongsideTransition: nil) { _ in
+            self.updateAppearance(tokenSet: self.tokenSet)
         }
+    }
 
-        switch positionController.arrowDirection {
-        case .up:
-            arrowImageView.frame.origin.y = 0.0
-            backgroundView.frame.origin.y = arrowImageView.frame.maxY
-        case .down:
-            arrowImageView.frame.origin.y = bounds.height - arrowImageView.frame.height
-        case .left:
-            arrowImageView.frame.origin.x = 0.0
-            backgroundView.frame.origin.x = arrowImageView.frame.maxX
-        case .right:
-            arrowImageView.frame.origin.x = bounds.width - arrowImageView.frame.width
-        }
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
 
         updateAppearance(tokenSet: tokenSet)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        if previousTraitCollection?.preferredContentSizeCategory != traitCollection.preferredContentSizeCategory {
+            viewWillLayoutSubviews()
```

Comment:
i am bit surprise by this line of code. i would expect viewwilllayoutsubivew should get automatically called?